www-client/qutebrowser/qutebrowser-1.8.1.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
This is still EAPI-5 which has been deprecated already, so upgrade to 7 since it should work.

Reply:
Ok